services/meta/client.go

```diff
@@ -957,35 +957,6 @@ func (c *Client) updateAuthCache() {
 	c.authCache = newCache
 }
 
-// snapshot will save the current meta data to disk
-func snapshot(path string, data *Data) error {
-	file := filepath.Join(path, metaFile)
-	tmpFile := file + "tmp"
-
-	f, err := os.Create(tmpFile)
-	if err != nil {
-		return err
-	}
-	defer f.Close()
-
-	var d []byte
-	if b, err := data.MarshalBinary(); err != nil {
-		return err
-	} else {
-		d = b
-	}
-
-	if _, err := f.Write(d); err != nil {
-		return err
-	}
-
-	if err = f.Close(); nil != err {
-		return err
-	}
-
-	return os.Rename(tmpFile, file)
```

Comment:
Include the conditional compilation flags like as it was done here: https://github.com/influxdata/influxdb/blob/master/tsdb/engine/tsm1/file_unix.go

This is also where I'm getting the naming standard from. So there's consistency between these files (as this file is also used for a similar reason to your issue), please use the basename `file` so there will be a `file_windows.go` and a `file_unix.go` and include the conditional compilation headers at the beginning. That should work.